src/Umbraco.Tests.Common/Builders/ConfigurationModelBuilders/ContentSettingsBuilder.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Core.Configuration.Models;
+using Umbraco.Core.Macros;
+
+namespace Umbraco.Tests.Common.Builders
```

Comment:
Not sure I see the value of the builder, when ContentSettings  has valid and good default values and is a poco?

Reply:
I don't actually see the benefit in ANY builders. The whole point of using `Options<T>` is that they are injected directly into services. 

Generally you want each type to be as simple as possible with sensible defaults so that external configuration covers edge case use only.

Reply:
It's partly to help with creating invalid setups so we can unit test that the validators work (see e.g. ContentSettingsValidationTests).  And partly just for consistency with what's gone before (we have builders for most of the configuration models - so probably we should have for all or none of them).

Reply:
Seems like a lot of ceremony. 

In fitting with ASP.NET Core I would build something around a  `ConfigurationBuilder` that parses a json source and creates an `IConfiguration` instance from that source. Much like the approach I demoed to create test configuration instances in #8654

Reply:
I still sees alot of benefits of some of the builders, just not for configuration (or any other POCO with default ctor and good default values).
E.g. `DataTypeBuilder` still makes it easier to create DataTypes.

Not sure I see the benefits of the builder in `ContentSettingsValidationTests` as you still use a "builder" method too. That method could just be replaced with this right?

```cs
 private static ContentSettings BuildContentSettings(string macroErrors = "inline", string culture = "en-US", string contentXPath = "", string autoFillImagePropertyAlias = "testAlias")
        {
            return new ContentSettings()
            {
                MacroErrors = macroErrors,
                Error404Collection = new[]
                {
                    new ContentErrorPage()
                    {
                        Culture = culture,
                        ContentId = 1,
                        ContentXPath = contentXPath
                    },
                },
                Imaging = new ContentImagingSettings()
                {
                    AutoFillImageProperties = new ImagingAutoFillUploadField[]
                    {
                        new ImagingAutoFillUploadField()
                        {
                            Alias = autoFillImagePropertyAlias
                        },
                    }
                }
            };
        }
```

Reply:
OK, I'll at an update later or tomorrow to remove the builders we've introduced for these configuration models.  Wasn't really seeing why they were considered a good idea for domain model classes, but not for these, but I see the distinction you are drawing, and makes sense.

Reply:
Updated now to that effect.